addon/components/one-way-select.js

```diff
@@ -0,0 +1,118 @@
+import Ember from 'ember';
+import layout from '../templates/components/one-way-select';
+
+import { invokeAction } from 'ember-invoke-action';
+
+const {
+  computed,
+  computed: { alias },
+  get,
+  set,
+  String: { w }
+} = Ember;
+
+export default Ember.Component.extend({
+  layout,
+  tagName: 'select',
+
+  attributeBindings: [
+    'autofocus',
+    'disabled',
+    'form',
+    'multiple',
+    'name',
+    'required',
+    'size'
+  ],
+
+  didReceiveAttrs() {
+    this._super(...arguments);
+
+    let options = this.getAttr('options');
+    if (typeof options === 'string') {
+      options = w(options);
+    }
+
+    set(this, 'options', Ember.A(options));
+  },
+
+  optionGroups: computed('options.[]', function() {
+    const groupLabelPath = get(this, 'groupLabelPath');
```

Comment:
Prefer not to use this kind of alignment, makes it annoying to add new variables